src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,18 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private Random random;
```

Comment:
```suggestion
    private Random random = new Random();
```
якщо в конструкторі ми не приймаємо значення для ініціалізації можна одразу через поле це робити без конструктора